container-aws-csharp/${PROJECT}.csproj

```diff
@@ -8,8 +8,8 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.*" />
-    <PackageReference Include="Pulumi.Aws" Version="5.*" />
-    <PackageReference Include="Pulumi.Awsx" Version="1.*" />
+    <PackageReference Include="Pulumi.Aws" Version="6.*" />
+    <PackageReference Include="Pulumi.Awsx" Version="2.0.0-beta.3" />
```

Comment:
Ah this is hard. We're changing this away from beta before merging yes? I feel like this is so easy to let slip through.